### DISCUSSION TITLE
请问播放视频的时后可以自动读取字幕吗

### DISCUSSION THREAD

**Discussion by kugimiya530** [AlistGo/alist#3016] [Q&A]
同级目录下
我放了跟视频同样檔名的ass但是读取不到

请问内建播放器  或是 另外呼叫potplayer有可能自行读取字幕吗

然后字幕大小能改吗....  内建的读取srt放大视频后字幕变很小

==========更新===========
A视频读取的到srt字幕

B视频读取不到srt字幕

两者都读取不到ass....这是啥情况

**Comment by rakaki** [comment]
还是用webdav吧省事
